src/style.css

```diff
@@ -2,2 +1,85 @@
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+li,
+a {
+  text-decoration: none;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.logo-link {
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+}
+
+.nav {
+  margin-left: auto;
+}
+
+.nav_links {
+  list-style: none;
+  height: 100%;
+}
+
+.nav_links li {
+  display: inline-block;
+  margin-right: 16px;
+  height: 100%;
```

Comment:
Be consistent with your *vertical* margins (Add only top or only bottom margin, but not both). Vertical margins can be collapsed in some cases.